### ISSUE TITLE
Unloaded tabs are not synced on Kiwi

### ISSUE THREAD

**Issue by sevmonster** [floccusaddon/floccus#1032]
### Describe the bug
Unloaded tabs (e.g. automatically unloaded to save memory or after browser launch) in Kiwi are not synced using tab sync.

### To Reproduce
1. Set up tab sync
2. Create new tabs and get them to unload (e.g. reopen Kiwi)
3. Start a push sync
4. Only active tabs appear

### Expected behavior
All tabs should be synced whether unloaded or not.

##### Desktop
 - OS: Android
 - Browser: Kiwi
 - Browser Version: 98.0.4758.60
 - Floccus version: 4.9.0
 - Floccus sync method: Nextcloud Bookmarks

### Server
 - OS: Alpine Linux Edge
 - Nextcloud version: 23.0.1
 - Bookmarks app version: 10.0.3

### Debug log
>It is often useful to provide a debug log file along with the issue. You can obtain a debug log of the most recent sync runs by clicking 'Debug log' in the bottom right corner of the floccus panel in your browser.

There is no button, and no sync error occured.

- [ ] Debug log provided

**Comment by marcelklehr** [comment]
Could not reproduce this with the latest version. (I opened a new tab in the background to test, maybe not enough?)

**Comment by marcelklehr** [comment]
I could reproduce with newly opening Kiwi, there's not much I can do about this behavior, though.

**Comment by marcelklehr** [comment]
Is this a duplicate of #1030 though?

**Comment by sevmonster** [comment]
I don't believe so, in this particular instance it's just the tabs not being included in the sync at all. I don't know if this behavior occurs on browsers with unloaded tabs. Maybe it's related somehow.

**Comment by sevmonster** [comment]
On a fresh install, I had 2 windows open and tabs kept moving between 
windows, duplicating, disappearing... I have to leave so I can't yet do 
any more thorough testing right now.

----------------------------------------

*From: *Marcel Klehr ***@***.***>
*To: *floccusaddon/floccus ***@***.***>
*CC: *sev ***@***.***>; Author ***@***.***>
*Date: *Mar 15, 2022 06:16:28
*Subject: *Re: [floccusaddon/floccus] Unloaded tabs are not synced on 
Kiwi (Issue #1032)

**Comment by qubist-pixel-ux** [comment]
Any update on this issue ? It makes floccus unusable. Tabs will be removed from sync file once unloaded by browser.

Do other browsers have this issue ?

**Comment by marcelklehr** [comment]
Hi @qubist-pixel-ux 
The status of this issue is that I cannot do anything to fix this. Unloaded tabs in Kiwi are not visible to floccus anymore, so there's nothing I can do in floccus to work around this.

**Comment by qubist-pixel-ux** [comment]
Are there any android browsers in which flocus works well ?


________________________________

**Comment by marcelklehr** [comment]
Not that I know of, besides, Kiwi. That's why the android app exists.
